lib/model/model.go

```diff
@@ -1298,6 +1300,12 @@ func (m *model) ccHandleFolders(folders []protocol.Folder, deviceCfg config.Devi
 				l.Warnf("Failed to persist pending folder entry to database: %v", err)
 			}
 			indexSenders.addPending(cfg, ccDeviceInfos[folder.ID])
+			updatedPending = append(updatedPending, map[string]string{
+				"folder":      folder.ID,
+				"folderLabel": folder.Label,
```

Comment:
I think I'd have either `id, label` or `folderId, folderLabel`,

Reply:
`id` might be confused with the event's sequence ID.  It's otherwise only used in `DeviceConnected` and `DeviceDisconnected` so far.

`folderID` is used once in `LocalChangeDetected`.

Most other event types agree on `device` and `folder` fields for the IDs. Should we try to stay consistent to that, or set a new standard starting here?

Reply:
`folder` in JS sometimes means ID, but there's also places where it's the entire config. Using `folderId` removes that ambiguity and that outweighs consistency with other events to me (respectively we might want to add a `folderId` field to those as well - not this PR though).

Reply:
Okay, will change. For consistency, it should be `folderID` and `deviceID` though, with all-caps ID?